PluginsAndFeatures/AddLibrary/AzureLibraries/com.microsoft.azuretools.sdk/dependencies/ThirdPartyNotices.txt

```diff
@@ -43,7 +43,7 @@ Section A: Microsoft is offering you a license to use the following components w
 35.	Retrofit (https://github.com/square/retrofit)
 36.	RxJava (https://github.com/ReactiveX/RxJava)
 37.	Simple Logging Facade for Java (https://www.slf4j.org/)
-
+38. HtmlUnit (http://htmlunit.sourceforge.net/)
```

Comment:
Do we need to append the open source license of `HtmlUnit` to the end of this document?